bridge_adaptivity/module/models.py

```diff
@@ -184,16 +192,13 @@ class Meta:
     def __str__(self):
         return "Engine: {}".format(self.engine_name)
 
-    def save(self, *args, **kwargs):
-        utils.save_model_parameter_true_once(self, 'is_default')
-        super(Engine, self).save(*args, **kwargs)
-
     @classmethod
-    def get_default_engine(cls):
-        return (
-            Engine.objects.filter(is_default=True).first() or
-            Engine.objects.get_or_create(engine=cls.DEFAULT_ENGINE, engine_name='Mock', is_default=True)[0]
-        )
+    def get_or_create_default(cls):
```

Comment:
default class instance is got in the get_default mixin's method it seems we need only create_default here.